mobify/platform-scaffold
# Add spinner to place order button

Added a spinner to `Place Order` button in checkout-payment container to aviod multiple form submittions.

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1404

## Changes
- Disable and show spinner while the payment is being submitted.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add anything to the cart and proceed to checkout
- In payment page, click `Place Order` button to test the loading state.
- Verify both statically placed button and fixed position button have the same behavior

## Result
![spinner](https://user-images.githubusercontent.com/10948652/27106262-767cfcb2-5047-11e7-971e-7be30a082499.gif)
